src/common/components/hosts/DiskRole.tsx

```diff
@@ -81,9 +82,9 @@ const DiskRoleDropdown: React.FC<DiskRoleDropdownProps> = ({
         // TODO(mlibra): Improve for the case onDiskRole === undefined
         setOpen(false);
       };
-      asyncFunc();
+      void asyncFunc();
```

Comment:
why is this needed?

Reply:
https://github.com/openshift-assisted/assisted-ui-lib/pull/936#discussion_r740035500

Reply:
I missed the response above. IMHO, it brings noise in the code.
But I would be ok with merging it.